src/pages/certifications/front-end-libraries/react/add-comments-in-jsx/index.md

```diff
@@ -2,9 +2,22 @@
 title: Add Comments in JSX
 ---
 ## Add Comments in JSX
+You can easily using JS comment but they need to wrapped by curly braces to add comments in JSX:
```

Comment:
Hi @danhmanh.

This PR is great :star2:.

This first sentence doesn't make sense. Did you mean to say that you can _*add*_ JavaScript comments? It seems that might be missing from the sentence. Can you please make the appropriate change and I'll review and merge it?
> You can easily using JS comment but they need